portfolio/src/main/webapp/style.css

```diff
@@ -56,6 +57,44 @@ p {
   color: white;
 }
 
+#login {
+  background-color: #F4EBD9;
```

Comment:
nit: Looks like some color hex code are uppercase and some of them are lower cases. Could you make them consistent?